test/e2e/node/kubelet_authz.go

```diff
@@ -104,6 +104,11 @@ func runKubeletAuthzTest(ctx context.Context, f *framework.Framework, endpoint,
 
 	err = e2eauth.WaitForAuthzUpdate(ctx, f.ClientSet.AuthorizationV1(),
 		serviceaccount.MakeUsername(ns, saName),
+		[]string{
+			"system:authenticated",
+			"system:serviceaccounts",
```

Comment:
Better to use `serviceaccount.MakeGroups(ns)` instead, but that doesn't include system:authenticated. Do you need that group? If so, maybe it can be include in `MakeGroups`?

Reply:
Fixed to use `serviceaccount.MakeGroups(ns)` and manually added `system:authenticated`. It doesn't seem like that group fits well in the return value of `serviceaccount.MakeGroups()`, as that function seems to be related to SA groups only.

Unfortunately I do need that group, and it seems reasonable to add it here as it present in real requests, since it's added in the authentication layer and is always present when the user is authenticated:

https://github.com/kubernetes/kubernetes/blob/e36a168a63039ff76e659a7ea2a91798d65c9f82/staging/src/k8s.io/apiserver/pkg/authentication/group/authenticated_group_adder.go#L56